package.json

```
@@ -14,6 +14,7 @@
     "dev:adapter-next": "yarn workspace @slicemachine/adapter-next dev",
     "dev:adapter-sveltekit": "yarn workspace @slicemachine/adapter-sveltekit dev",
     "dev:e2e-next": "cd ./e2e-projects/next && yarn dev --port 8000",
+    "dev:init": "yarn workspace @slicemachine/init dev",
```

nitpick: I just feel like it makes more sense to have all "packages/workspace" scripts to be grouped together, then the e2e ones~

```suggestion
    "dev:init": "yarn workspace @slicemachine/init dev",
    "dev:e2e-next": "cd ./e2e-projects/next && yarn dev --port 8000",
```

---

I agree with the suggestion. It's fixed now.